frontend/src/components/regions/CardDetailsModal/Presentation.js

```diff
@@ -122,6 +126,7 @@ const CardBasicDetails = ({ card }) => {
 
 export default ({
   card,
+  authUser,
```

Comment:
Now the CardDetailsModal requires that it has access to the authUser.

Does it have access to the authUser? I dont think anything passes authUser to this component.

If you have a function like this:

```
function importantStuff({something}){
   ....stuff
}

```

And you change it to:

```
function importantStuff({something,and,more,stuff, authUser}){
   ....stuff
}
```

But it gets called like this in other areas of the code base:

```
importantStuff({something})
```

Will it work? 

No. It needs to be called with the correct arguments.



If a Presentation component needs an argument, then that argument needs to be passed in. Otherwise it wont work. The component will break as soon as we try to actually use it in real life. 

## what to do:

For each component you'll see that there are at least 2 files:

Presentation.js <- this has the looks
index.js <- this has the brains. This renders the presentation component with the correct props.

If you change the presentation component so it needs extra props, then make sure the index.js component actually sends the right props.